icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/RbnfTest.java

```diff
@@ -820,6 +820,35 @@ public void TestFractionalRuleSet() {
         doTest(formatter, testData, false); // exact values aren't parsable from fractions
     }
 
+    @Test
+    public void TestNorwegianSpellout()
+    {
+        Locale noLocale = new Locale("no", "", "");
+        Locale nbLocale = new Locale("nb", "", "");
+        RuleBasedNumberFormat noFormatter = new RuleBasedNumberFormat(noLocale,
+                RuleBasedNumberFormat.SPELLOUT);
+        RuleBasedNumberFormat nbFormatter = new RuleBasedNumberFormat(nbLocale,
+                RuleBasedNumberFormat.SPELLOUT);
+
+        String[][] testDataDefault = {
+                { "1", "\u00E9n" },
```

Comment:
in java, why escapes rather than real characters?

in c++, i see that the shared test function takes `const char *` which may not be unicode, but in java we have no such constraint.

Reply:
I was following the existing precedent. I imagine the reason is to make it easier to copy data from the C test to Java, or vice versa (since these same tests exist in both.